### ISSUE TITLE
TFLiteConverter - GetOpWithOutput error

### ISSUE THREAD

**Issue by Pierrci** [tensorflow/tensorflow#34210]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Google Colab
- TensorFlow installed from (source or binary): `pip install tf-nightly`
- TensorFlow version (use command below): 2.1.0-dev20191111
- Python version: 3.6.8

**Describe the current behavior**
I'm trying to convert a BERT keras model imported from the `transformers` library. Using the `convert()` method from the `TFLiteConverter`, the following `GetOpWithOutput` error happens, no matter the values used passed to `allow_custom_ops` or `target_spec.supported_ops`:

`tensorflow/lite/toco/tooling_util.cc:935] Check failed: GetOpWithOutput(model, output_array) Specified output array "Identity" is not produced by any op in this graph`

**Describe the expected behavior**
The TFLiteConverter should convert the model and return the tflite version

**Code to reproduce the issue**
https://colab.research.google.com/drive/16kWs2ji8xrgjSuLftXknDqEh2VixDR4R

**Other info / logs**
The same `bert-large-uncased-whole-word-masking-finetuned-squad` model imported from the `transformers` library works perfectly when used directly on a QA task without TFLite conversion.

Also, the conversion of the DistilBERT model from the same library and using the same steps works correctly with `converter.target_spec.supported_ops = [tf.lite.OpsSet.SELECT_TF_OPS]`.

**Comment by haozha111** [comment]
Hi,

It's possible that the model doesn't have an output node named "Identity". Can you use netron (https://lutzroeder.github.io/netron/) to visualize this graph and see what's the output?

(I'm guessing this is just a tensor naming issue, if not I can take a further look)

**Comment by Pierrci** [comment]
Thank you for your answer. I tried to visualize the graph using Netron without success, either with a `h5` or `SavedModel` version.

However, as you can see in [this cell of the notebook](https://colab.research.google.com/drive/16kWs2ji8xrgjSuLftXknDqEh2VixDR4R#scrollTo=94UoamBP17VO&line=4&uniqifier=1), it seems there are two outputs `Identity` and `Identity_1`. Also visible in this Tensorboard graph:
<img width="751" alt="Screen Shot 2019-11-13 at 2 34 35 PM" src="https://user-images.githubusercontent.com/5020707/68797525-c1e6bb00-0622-11ea-99ef-b5c1e8454ccc.png">

Does it answer your question @haozha111?

**Comment by haozha111** [comment]
That seems like a bug in our old converter.

We have developed a new MLIR-based converter, you can try it out by downloading the tf-nightly pip package, and then set converter.experimental_new_converter = True.

Could you try this out and see if the problem is resolved?

**Comment by Pierrci** [comment]
I get this error when trying with `converter.experimental_new_converter = True`, whatever the value of other options `target_spec.supported_ops` and `allow_custom_ops`:

```
2019-11-15 15:18:32.489426: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/lib64-nvidia
2019-11-15 15:18:34.411116: W tensorflow/compiler/mlir/lite/python/graphdef_to_tfl_flatbuffer.cc:106] Ignored output_format.
2019-11-15 15:18:34.411167: W tensorflow/compiler/mlir/lite/python/graphdef_to_tfl_flatbuffer.cc:112] Ignored drop_control_dependency.
Traceback (most recent call last):
  File "/usr/local/bin/toco_from_protos", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/dist-packages/tensorflow_core/lite/toco/python/toco_from_protos.py", line 93, in main
    app.run(main=execute, argv=[sys.argv[0]] + unparsed)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow_core/python/platform/app.py", line 40, in run
    _run(main=main, argv=argv, flags_parser=_parse_flags_tolerate_undef)
  File "/usr/local/lib/python3.6/dist-packages/absl/app.py", line 299, in run
    _run_main(main, args)
  File "/usr/local/lib/python3.6/dist-packages/absl/app.py", line 250, in _run_main
    sys.exit(main(argv))
  File "/usr/local/lib/python3.6/dist-packages/tensorflow_core/lite/toco/python/toco_from_protos.py", line 56, in execute
    enable_mlir_converter)
Exception: Graph does not contain node: input_1
```

But it seems there is an `input_1` in the model, as visible in my previous TensorBoard graph and in [this cell of the notebook](https://colab.research.google.com/drive/16kWs2ji8xrgjSuLftXknDqEh2VixDR4R#scrollTo=94UoamBP17VO&line=4&uniqifier=1).

**Comment by haozha111** [comment]
I could reproduce the issue now. It seems like a bug in our converter, will take a look and try to fix it soon.

**Comment by ucalyptus** [comment]
I changed `transformers>=2` in the pip command. https://colab.research.google.com/drive/1MrPEJWqReZGKWBHAkoYYCrDiDTccmLUb
Does this help?

**Comment by Pierrci** [comment]
I tried again with the latest TF nightly version and it works now using the experimental converter and partial TF operators:

```python
converter.experimental_new_converter = True
converter.target_spec.supported_ops  = [tf.lite.OpsSet.TFLITE_BUILTINS, tf.lite.OpsSet.SELECT_TF_OPS]
```

Specifying `transformers>=2` doesn't affect the result since it defaults to the last version available which is `2.3.0` @ucalyptus.

Closing this, thanks to the TF team for all the continuous improvements! 👏

**Comment by alanrabelo** [comment]
Using these options did great! Thanks
**converter.experimental_new_converter = True
converter.allow_custom_ops = True**
